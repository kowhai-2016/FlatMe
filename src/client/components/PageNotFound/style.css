.Pnf {
  background-image: url('/images/crazy-man.jpg');
  min-height: 800px;
  background-repeat: no-repeat;
  background-position: center;
}

.bg-pnf h1 {
  margin-left: 13%;
  margin-top: 13%;
  font-size: 3em;
}

@media screen and (max-width: 767px) and (min-width: 200px) {
  .bg-pnf h1  {
    margin-top: 10%;
    font-size: 1.5em;
  }
}
