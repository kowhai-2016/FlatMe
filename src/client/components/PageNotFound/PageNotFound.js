import React from 'react'

import './style.css'

const PageNotFound = props => {
  return (
    <div className='Pnf container-fluid'>
      <div className='row'>
        <div className='col-md-12 bg-pnf'>
          <h1>404, Did you <br />lost your way ?</h1>
        </div>
      </div>
    </div>
  )
}

export default PageNotFound
