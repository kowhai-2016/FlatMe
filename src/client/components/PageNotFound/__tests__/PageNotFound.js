/* eslint-env jest */

import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'

import PageNotFound from '..'

test('PageNotFound renders correctly', () => {
  const wrapper = shallow(<PageNotFound />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})
