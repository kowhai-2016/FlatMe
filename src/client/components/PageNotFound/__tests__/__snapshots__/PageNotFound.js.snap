exports[`test PageNotFound renders correctly 1`] = `
<div>
  <h1>
    PageNotFound
  </h1>
</div>
`;
